accessors/bq/doc/REFERENCE.md

```diff
@@ -0,0 +1,26 @@
+## accessors
+
+<div class="badge core"></div>
+
+TODO: module description.
```

Comment:
Missing description and documentation of `ST_ENVELOPE`

Reply:
True missed, totally ^^'. Going for it

Reply:
Cool!